.github/ISSUE_TEMPLATE/translation_issue.yml

```diff
@@ -1,79 +1,117 @@
-  validations:
-    required: true
+  - type: checkboxes
+    id: preparation
+    attributes:
+      label: Before filing a new issue, please check the following
+      description: This makes it easier for us all :)
+      options:
+        - label: I am using the latest version (check for updates in Settings)
+          required: true
+        - label: I have searched in [open issues](https://github.com/microsoft/PowerToys/issues) (remember to try different search terms)
+          required: true
+        - label: I have searched in [closed issues](https://github.com/microsoft/PowerToys/issues?q=is%3Aissue+is%3Aclosed) (there might be a good reason)
+          required: true
+
+  - type: input
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: Microsoft PowerToys version
+      description: Hover over the system tray icon or look at Settings
+      placeholder: 0.80.0
+
+  - type: dropdown
+    id: area
+    validations:
+      required: true
+    attributes:
+      label: Area with translation issue
+      options:
+        - Always on Top
+        - Awake
+        - ColorPicker
+        - Command not found
+        - Crop and Lock
+        - Environment Variables
+        - FancyZones
+        - FancyZones Editor
+        - File Locksmith
+        - "File Explorer: Preview Pane"
+        - "File Explorer: Thumbnail preview"
+        - Hosts File Editor
+        - Image Resizer
+        - Installer
+        - Keyboard Manager
+        - Mouse Utilities
+        - Mouse Without Borders
+        - Paste as Plain Text
+        - Peek
+        - PowerRename
+        - PowerToys Run
+        - Quick Accent
+        - Registry Preview
+        - Screen ruler
+        - Settings
+        - Shortcut Guide
+        - System tray interaction
+        - TextExtractor
+        - Video Conference Mute
+        - Welcome / PowerToys Tour window
+        - Other
+
+  - type: dropdown
+    id: language
+    validations:
+      required: true
+    attributes:
+      label: Language
+      options:
+        - Arabic (Saudi Arabia)
+        - Chinese (simplified)
+        - Chinese (traditional)
+        - Czech
+        - Dutch
+        - English
+        - French
+        - German
+        - Hebrew
+        - Hungarian
+        - Italian
+        - Japanese
+        - Korean
+        - Persian
+        - Polish
+        - Portuguese
+        - Portuguese (Brazil)
+        - Russian
+        - Spanish
+        - Turkish
+        - Ukrainian
+
+  - type: textarea
+    id: displayedtext
+    validations:
+      required: true
+    attributes:
+      label: ❌ Displayed text
+      description: Please include a screenshot as that is extremely helpful.
+
+  - type: textarea
+    id: correcttext
+    validations:
+      required: true
+    attributes:
+      label: ✔️ Correct text
+
+  - type: textarea
+    id: context
+    validations:
+      required: true
+    attributes:
+      label: ℹ Context, explanation
```

Comment:
the prior body here was more user friendly,  i prefer the other tone.  please revert